tools/state-viewer/src/commands.rs

```diff
@@ -841,3 +844,33 @@ fn format_hash(h: CryptoHash, show_full_hashes: bool) -> String {
 pub fn chunk_mask_to_str(mask: &[bool]) -> String {
     mask.iter().map(|f| if *f { '.' } else { 'X' }).collect()
 }
+
+pub(crate) fn contract_accounts(
+    home_dir: &Path,
+    store: Store,
+    near_config: NearConfig,
+) -> anyhow::Result<()> {
+    let (_runtime, state_roots, _header) = load_trie(store.clone(), home_dir, &near_config);
+
+    for (shard_id, &state_root) in state_roots.iter().enumerate() {
+        eprintln!("Starting shard {shard_id}");
```

Comment:
I can see the consistency argument but I believe `eprintln` is the right choice here anyway.

The reason this goes to stderr is precisely because I don't want it in redirected output. This is just a general progress update to reassure whoever sits in front of the terminal that things are moving along. But scripts working with the output will not want that line.

Case in point, in [this comment](https://github.com/near/nearcore-private/issues/1#issuecomment-1384567730) I have outlined a number of in-terminal data transformations. The script below that adds up all sizes would actually break if there was a line like the above, where the second field is not a number.

```bash
awk '{total+=$2} END{printf "%d\n", total}' < contract_accounts.log 
```

Reply:
Oh but if you still think it's better for consistency to not write to `eprintln!`, then I can also just remove this line, I don't care about it that much. As long as it doesn't go to stdout. :)

Reply:
I see your point and it definitely makes sense to print progress messages to stderr.

Let's keep `eprintln!` here, I think the right solution would be to fix all other instances of printing progress messages to stdout (but definitely not a part of this PR).